doc/team.md

```diff
@@ -22,78 +22,64 @@ responsibilities. Distinguished Contributors who are also active maintainers of
-        <div class="team-card-body">
-            <h6 class="team-card-text">Michael Grund</h6>
-            <h7 class="team-card-text">Maintainer</h7>
-            <h7class="team-card-text"><a href="https://github.com/michaelgrund" >@michaelgrund</a></h7>
-        </div>
-    </div>
-    <div class="team-card">
-        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/26473567?v=4" alt="Card image">
-        <div class="team-card-body">
-            <h6 class="team-card-text">Paul Wessel</h6>
-            <h7 class="team-card-text">Founder</h7>
-            <h7class="team-card-text"><a href="https://github.com/PaulWessel" >@PaulWessel</a></h7>
-        </div>
-    </div>
-    <div class="team-card">
-        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/23487320?v=4" alt="Card image">
-        <div class="team-card-body">
-            <h6 class="team-card-text">Wei Ji Leong</h6>
-            <h7 class="team-card-text">Maintainer</h7>
-            <h7class="team-card-text"><a href="https://github.com/weiji14" >@weiji14</a></h7>
-        </div>
-    </div>
-    <div class="team-card">
-        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/29518865?v=4" alt="Card image">
-        <div class="team-card-body">
-            <h6 class="team-card-text">Will Schlitzer</h6>
-            <h7 class="team-card-text">Maintainer</h7>
-            <p class="team-card-text"><a href="https://github.com/willschlitzer" >@willschlitzer</a></h7>
-        </div>
-    </div>
-    </div>
-```
+:::::{grid} 2 3 3 4
+::::{grid-item-card} Dongdong Tian
+:img-top: https://avatars.githubusercontent.com/u/3974108?v=4
+
+Maintainer
+[@seisman](https://github.com/seisman)
+::::
+
+::::{grid-item-card} Jiayuan Yao
+:img-top: https://avatars.githubusercontent.com/u/50591376?v=4
+
+[@core-man](https://github.com/core-man)
+::::
+
+::::{grid-item-card} Leonardo Uieda
+:img-top: https://avatars.githubusercontent.com/u/290082?v=4
+
+Founder
+[@leouieda](https://github.com/leouieda)
+::::
+
+::::{grid-item-card} Liam Toney
+:img-top: https://avatars.githubusercontent.com/u/38269494?v=4
+
+[@liamtoney](https://github.com/liamtoney)
+::::
+
+::::{grid-item-card} Max Jones
+:img-top: https://avatars.githubusercontent.com/u/14077947?v=4
+
+Maintainer
+[@meghanrjones](https://github.com/meghanrjones)
+::::
+
+::::{grid-item-card} Michael Grund
+:img-top: https://avatars.githubusercontent.com/u/23025878?v=4
+
+Maintainer
+[@michaelgrund](https://github.com/michaelgrund)
+::::
+
+::::{grid-item-card} Paul Wessel
+:img-top: https://avatars.githubusercontent.com/u/26473567?v=4
+
+Founder
+[@PaulWessel](https://github.com/PaulWessel)
+::::
+
+::::{grid-item-card} Wei Ji Leong
+:img-top: https://avatars.githubusercontent.com/u/23487320?v=4
+
+Maintainer
+[@weiji14](https://github.com/weiji14)
```

Comment:
```suggestion
[@weiji14](https://github.com/weiji14)
+++
Maintainer
```
| Before | After |
|-------|------|
| 
<img width="208" alt="image" src="https://user-images.githubusercontent.com/14077947/171254567-ef460eea-c895-4541-b786-35200e3f1020.png"> | <img width="203" alt="image" src="https://user-images.githubusercontent.com/14077947/171254469-32b5bd37-9877-4a7b-9476-a9ebe9c4bbe2.png"> |

Reply:
Good suggestion! I've used the footer option for labels and also use [sphinx-design's badge directive](https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html) to highlight different roles.

![image](https://user-images.githubusercontent.com/3974108/171318962-180faa18-db8a-4695-9766-b39e81c5c0a6.png)